client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx

```diff
@@ -60,9 +66,14 @@ const MarketplacePluginInstall = ( {
 		getUploadedPluginId( state, siteId )
 	) as string;
 	const pluginUploadComplete = useSelector( ( state ) => isPluginUploadComplete( state, siteId ) );
-	const installedPlugin = useSelector( ( state ) =>
```

Comment:
For some reason this cannot find the correct type from the selector, so we hard-code it here by starting from `unknown` and using type guards.

Reply:
Turns out this did not work if the selector returned undefined. https://github.com/Automattic/wp-calypso/pull/59608 should fix it in a safer way.